packages/slate/src/transforms/general.ts

```diff
@@ -167,6 +167,7 @@ export const GeneralTransforms: GeneralTransforms = {
 
       case 'remove_text': {
         const { path, offset, text } = op
+        if (!text || text.length === 0) break
```

Comment:
Can you remove the `!text` check, it's unnecessary since `text` seems to always be a string here.

Reply:
Ok, I pushed a change for this.

I also added a similar check for 'insert_text'.